itest/rust/src/object_tests/object_test.rs

```diff
@@ -1076,3 +1076,59 @@ fn double_use_reference() {
     double_use.free();
     emitter.free();
 }
+
+// ----------------------------------------------------------------------------------------------------------------------------------------------
+
+#[derive(GodotClass)]
+#[class(init, base=Object)]
+struct MultipleImplBlocks {}
+
+#[godot_api]
+impl MultipleImplBlocks {
+    #[func]
+    fn foo(&self) -> String {
+        "result of foo".to_string()
+    }
+}
+
+#[godot_api(secondary)]
+impl MultipleImplBlocks {
+    #[func]
+    fn bar(&self) -> String {
+        "result of bar".to_string()
+    }
+}
+
+#[godot_api(secondary)]
+impl MultipleImplBlocks {
+    #[func]
+    fn baz(&self) -> String {
+        "result of baz".to_string()
+    }
+}
+
+/// Test that multiple inherent '#[godot_api]' impl blocks can be registered.
+/// https://github.com/godot-rust/gdext/pull/927
+#[itest]
+fn godot_api_multiple_impl_blocks_pull_927() {
+    let mut obj: Gd<MultipleImplBlocks> = MultipleImplBlocks::new_alloc();
+
+    fn call_and_check_result(
+        gd: &mut Gd<MultipleImplBlocks>,
+        method_name: StringName,
+        expected_result: String,
+    ) {
+        assert!(gd.has_method(&method_name));
+        let result = gd.call(&method_name, &[]);
+        let result_as_string = result.try_to::<String>();
+        assert!(result_as_string.is_ok());
+        assert_eq!(expected_result, result_as_string.unwrap());
```

Comment:
Better as one statement -- also, expected should be on the right:
```suggestion
        assert_eq!(result_as_string, Ok(expected_result));
```

Reply:
![image](https://github.com/user-attachments/assets/b11bd1e3-9452-4729-8286-6b2e59bbd810)

I've switched back to two lines for now